js/redoc.standalone.js

```diff
@@ -0,0 +1,1740 @@
+/*! For license information please see redoc.standalone.js.LICENSE.txt */
```

Comment:
Was `.standalone` the new name they use for this? Or was this file compiled by you? Does a minified version exist that we could use?

Reply:
Yeah, this is just taken from redoc. It looks minified already, apart from the line breaks for some reason.

Reply:
I was hoping to keep the same name, and just replace the file, but it is what it is then 😅 🤷‍♂️